posthog/models/organization.py

```diff
@@ -137,13 +137,13 @@ def validate(self, *, user: Optional[Any], email: Optional[str] = None) -> None:
             assert user is not None, "Either user or email must be provided!"
             email = user.email
         if email != self.target_email:
-            raise ValueError("Invite is not intended for this email.")
+            raise ValueError("Invite intended for another email address.")
         if OrganizationMembership.objects.filter(organization=self.organization, user=user).exists():
             raise ValueError("User already is a member of the organization.")
         if OrganizationMembership.objects.filter(
             organization=self.organization, user__email=self.target_email
         ).exists():
-            raise ValueError("Target email already is a member of the organization.")
+            raise ValueError("Email address already is a member of the organization.")
```

Comment:
I would change to something like "A user with this email address is already ....". Maybe tangential to this PR but we do validate this too when creating the invite right?